examples/cluster/plot_birch_vs_minibatchkmeans.py

```diff
@@ -35,7 +36,7 @@
 n_centres = np.hstack((np.ravel(xx)[:, np.newaxis], np.ravel(yy)[:, np.newaxis]))
 
 # Generate blobs to do a comparison between MiniBatchKMeans and BIRCH.
-X, y = make_blobs(n_samples=100000, centers=n_centres, random_state=0)
+X, y = make_blobs(n_samples=25000, centers=n_centres, random_state=0)
```

Comment:
We could also rename `n_centres` to `n_centers` while we are at it.

Reply:
Done with the last commit.